pandas_profiling/utils/notebook.py

```diff
@@ -6,3 +6,13 @@ def full_width():
     from IPython.core.display import display, HTML
 
     display(HTML("<style>.container { width:100% !important; }</style>"))
+
+
+def in_ipynb() -> bool:
```

Comment:
I've not found a specific method to do this in the another modules, so I decided to add it. Let me know if there's a better approach to detect the environment.

Reply:
> It is also possible to let tqdm automatically choose between console or notebook versions by using the autonotebook submodule:
> ```from tqdm.autonotebook import tqdm```

https://github.com/tqdm/tqdm